R/RLearner_classif_randomForest.R

```diff
@@ -50,3 +50,28 @@ predictLearner.classif.randomForest = function(.learner, .model, .newdata, ...)
   type = ifelse(.learner$predict.type=="response", "response", "prob")
   predict(.model$learner.model, newdata = .newdata, type = type, ...)
 }
+
+getFeatureImportance.classif.randomForest = function(.learner, .model, ...) {
```

Comment:
Not sure if I understand you here. So we have parameters that are required to be a certain value (e.g. for ranger `importance = "impurity"` or `"permutation"`). And we should set an additional flag in the learner when the conditions to calculate variable importance are fullfilled, so that when we actually call `getFeatureImportance` we can just assert the flag?